lib/phlex/rails/form.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module Phlex
+	module Rails
+		module Form
+			def initialize(model)
+				@model = model
+				@url = @view_context.url_for(@model)
+			end
+
+			def self.input_field(method_name, type:)
```

Comment:
Is there a reason why this method isn't named or aliased to `field`?
```suggestion
			def self.field(method_name, type:)
```